tests/cephadm/test_cephadm_upgrade.py

```diff
@@ -3,10 +3,88 @@
 
 """
 import logging
+from concurrent.futures import ThreadPoolExecutor
+from time import time
 
 from ceph.ceph_admin.orch import Orch
 
-LOG = logging.getLogger()
+LOG = logging.getLogger(__name__)
+
+
+class RadosBench:
```

Comment:
i think this can lie some where outside this file. probably in utils or create a new rados.py file

Reply:
It is an individual entity hence it could be utils or lib.

Reply:
Could you add documentation about the object/class?

Reply:
> i think this can lie some where outside this file. probably in utils or create a new rados.py file

yeah, this part is in the plan now, I was just trying to utilize Radosbench for now. But definitely need to be part of libraries.
Need a plan for generic approach to handle this kind libraries.

-  Thank you.